packages/react-router/lib/types/register.ts

```diff
@@ -0,0 +1 @@
+export interface Register {}
```

Comment:
note to self: maybe this should go in `utils` so we don't have a one-line, net-new module